### ISSUE TITLE
It is possible to port a raspberry version?

### ISSUE THREAD

**Issue by XanaduNWH** [laborer/stcflash#13]
I want to burn a 8051chip use a raspberry without  usb adapter

It is possible?
thanks!

**Comment by laborer** [comment]
Yes.  Raspberry Pi 3 Model B has one available UART port, so you should be able to use it program a STC MCU without using USB.
